### ISSUE TITLE
UI Redesign - Mobile - Onboarding

### ISSUE THREAD

**Issue by ksanty** [atsign-foundation/atmosphere_pro#207]
**Description**
The task is to implement the new design for the onboarding menu in the mobile app.

**Assets and Design Screens**
All assets and design screens have already been created.

- The Figma design can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=2%3A3
- The Assets required can be found here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=0%3A1

For help with using Figma please watch this short tutorial video
https://user-images.githubusercontent.com/69275762/192322842-2ac40f88-c396-4a66-933b-3c8abae516c3.mov

**Additional context**
This work should be done on the UI_Redesign branch https://github.com/atsign-foundation/atmosphere_pro/tree/ui_redesign

**Comment by Wave780** [comment]
@ksanty I want this issues

**Comment by ksanty** [comment]
> @ksanty I want this issues

Great!  Let us know if you have any questions.

**Comment by Wave780** [comment]
Okay, I hope it's only the user interface

**Comment by ksanty** [comment]
> Okay, I hope it's only the user interface

It is only the UI which have already been redesigned.  @tinashe404 any other context here since he taking on an advanced task?

**Comment by Wave780** [comment]
I'm trying to understand the code base, I hope it is the welcome screen I can do the onboarding screen

**Comment by tinashe404** [comment]
@Wave780 This ticket pertains to the 7 screens which are in the Onboarding section in the Figma design here https://www.figma.com/file/gnhrfcg8OYvAz0zJXZz2r3/Atmosphere-Pro?node-id=2%3A3

**Comment by adilwahla** [comment]
would love to contribute in this issue we can split it @tinashe404 what do you think?

**Comment by Wave780** [comment]
I'm still on it, but you can split it, so I can know where to cover

**Comment by adilwahla** [comment]
cool let me know what you have covered so far?

**Comment by adilwahla** [comment]
total 6 of same kind we can split by 3? which one will you pick

**Comment by Wave780** [comment]
I will go for the first 3

**Comment by adilwahla** [comment]
cool 👍  we both should update to keep whole code refactored timely

**Comment by tinashe404** [comment]
> would love to contribute in this issue we can split it @tinashe404 what do you think?

Yes, feel free to collaborate in whatever way works best for you.

**Comment by adilwahla** [comment]
Thanks for assigning me 
I was thinking i should code for android & for iOS @Wave780 has already started. Just give me go ahead call for android :)
what are you thoughts!

**Comment by tinashe404** [comment]
@adilwahla sure. If @Wave780 is on board, please go ahead.

**Comment by adilwahla** [comment]
Thanks <3

**Comment by Wave780** [comment]
Can we be communicating here or do you have any other platform
